zmon-controller-app/src/main/resources/templates/index.html

```diff
@@ -99,13 +102,18 @@
     <script th:inline="javascript" th:if="${eumTracingEnabled}">
       var eumTracingConfig = /*[[${eumZmonTracingConfig}]]*/  { name: "noop", config: {} };
       eumTracingConfig = JSON.parse(eumTracingConfig);
+
+      // initialize app if OT takes longer than 2s to load and initialize
+      var timeout = setTimeout( () => bootstrap(), 2000);
+
       opentracingJavascriptUtils.initOpenTracing(eumTracingConfig)
                                     .then((e) => {
                                         console.log('Opentracing Initialized');
                                     }).catch( (e) => {
                                         console.log("Couldn't initialize Opentracing.", e)
```

Comment:
Should we clear the timeout here as well?

Reply:
no need, the following then() is executed after this catch() (or after the first then()). The second then() could actually be replaced by a finally() which would prob make it clearer.